lib/perl/Genome/Model/ClinSeq/Util.pm

```diff
@@ -1226,7 +1227,8 @@ sub get_ref_align_builds{
 
       $ref_builds{$refalign_name}{type} = $build_type;
       $ref_builds{$refalign_name}{sample_name} = $subject_name;
-      $ref_builds{$refalign_name}{sample_name_build_type} = $subject_name . "_" . $build_type;
+      $ref_builds{$refalign_name}{sample_name_build_type} =
+        $subject_name . "_" . $subject_common_name . "_" . $subject_icn . "_" . $build_type;
```

Comment:
Is it okay to have multiple consecutive `_`s in these headers if the subject's or the individual's common names are not set?  (Currently this will produce warnings if they are not.  It would be better to either default to the empty string or omit them as preferred.)

This could use `join('_', ...)` instead a series of concatenations.